### PR TITLE
New server localization strings

### PR DESCRIPTION
**Pull request by bakulf**

### CODE REVIEW

**Review by flodolo — APPROVED** [review]

File: tests/functional/localizeServers.js [hunk 1]
```diff
@@ -262,9 +275,19 @@ describe('Server list', function() {
     for (let lang of result.results.bindings) {
       const langCode = lang.countryName['xml:lang'];
       const value = lang.countryName['value'];
-      if (languages.has(langCode) && value != server.name) {
+
+      if (value === server.name) continue;
```
Comment (birdsarah) [inline tests/functional/localizeServers.js:279]: discussed offline - remove this check as then it is easier to test the data

**Review by birdsarah — APPROVED** [review]
Fine with my comment to change this data check later. I reviewed the outputted data and looks sane to me based on (a) the chinese translation is never missing (b) what I know about country/city names in other languages (limited).

Here is the outputs.

Missing country locales - these *should* be missing if they are spelled the same as the en-US version of name:

```

**Comment by birdsarah** [comment]
I think there is a bug still. If you look at the output for `Atlanta, GA` it is:

```
'city': 'Atlanta, GA',
  'languages': {'bg': 'Атланта',
   'co': 'Atlanta',
   'cy': 'Atlanta',
   'de': 'Atlanta',
   'el': 'Ατλάντα',
   'en': 'Atlanta',
   'en_CA': 'Atlanta',
   'en_GB': 'Atlanta',
   'es_AR': 'Atlanta',
   'es_ES': 'Atlanta',
   'es_MX': 'Atlanta',
   'fr': 'Atlanta',
   'fy_NL': 'Atlanta',
   'hu': 'Atlanta',
   'ia': 'Atlanta',
   'id': 'Atlanta',
   'it': 'Atlanta',
   'ja': 'アトランタ',
   'nl': 'Atlanta',
   'oc': 'Atlanta',
   'pt_BR': 'Atlanta',
   'ru': 'Атланта',
   'sk': 'Atlanta',
   'sq': 'Atlanta',
   'sv_SE': 'Atlanta',
   'uk': 'Атланта',
   'zh_TW': '亞特蘭大'}},
```

This has duplicated content in because "Atlanta" does not match "Atlanta, GA". But as we're going to follow-up to get rid of the de-duplication line. I think this is fine.

But, @bakulf, is it fine to display "Atlanta" not "Atlanta, GA". I think yes and that's intentional, but just checking.

**Comment by birdsarah** [comment]
A couple more samples from the US cities highlighting some differences in the data:

```
 {'city': 'San Jose, CA',
  'languages': {'bg': 'Сан Хосе',
   'cy': 'San Jose, Califfornia',
   'de': 'San José',
   'el': 'Σαν Χοσέ',
   'en': 'San Jose',
   'en_CA': 'San Jose, California',
   'en_GB': 'San Jose',
   'es_AR': 'San José',
   'es_ES': 'San José',
   'es_MX': 'San José',
   'fr': 'San José',
   'fy_NL': 'San Jose',
   'hu': 'San José',
   'ia': 'San Jose',
   'id': 'San Jose, California',
   'it': 'San Jose',
   'ja': 'サンノゼ',
   'nl': 'San Jose',
   'oc': 'San José',
   'pt_BR': 'São José',
   'ru': 'Сан-Хосе',
   'sk': 'San José',
   'sq': 'San Hose',
   'sv_SE': 'San Jos',
   'uk': 'Сан-Хосе',
   'zh_TW': '聖荷西'}},
 {'city': 'Seattle, WA',
  'languages': {'bg': 'Сиатъл',
   'co': 'Seattle',
   'cy': 'Seattle',
   'de': 'Seattle',
   'el': 'Σιάτλ',
   'en': 'Seattle',
   'en_CA': 'Seattle',
   'en_GB': 'Seattle',
   'es_AR': 'Seattle',
   'es_ES': 'Seattle',
   'es_MX': 'Seattle',
   'fr': 'Seattle',
   'fy_NL': 'Seattle',
   'hu': 'Seattle',
   'ia': 'Seattle',
   'id': 'Seattle',
   'it': 'Seattle',
   'ja': 'シアトル',
   'nl': 'Seattle',
   'oc': 'Seattle',
   'pt_BR': 'Seattle',
   'ru': 'Сиэтл',
   'sk': 'Seattle',
   'sq': 'Seattle',
   'sv_SE': 'Seattle',
   'uk': 'Сіетл',
   'zh_TW': '西雅圖'}},
 {'city': 'Secaucus, NJ',
  'languages': {'cy': 'Secaucus, New Jersey',
   'de': 'Secaucus',
   'en': 'Secaucus',
   'en_CA': 'Secaucus',
   'en_GB': 'Secaucus',
   'es_AR': 'Secaucus',
   'es_ES': 'Secaucus',
   'es_MX': 'Secaucus',
   'fr': 'Secaucus',
   'it': 'Secaucus',
   'ja': 'ニュージャージー州セコーカス',
   'nl': 'Secaucus',
   'pt_BR': 'Secaucus',
   'ru': 'Секокус (Нью-Джерси)',
   'uk': 'Сікокес',
   'zh_TW': '西考克斯市，新泽西州'}}]

```

**Pull request merged**